docs/se/scheduling.adoc

```diff
@@ -35,14 +50,14 @@ include::{rootdir}/includes/dependencies.adoc[]
 </dependency>
 ----
 
-== Scheduling
+== Usage
 For scheduling periodic tasks it is possible to choose fixed rate setup or Cron expression.
```

Comment:
Maybe better: "For scheduling periodic tasks, it is possible to choose a fixed rate or a Cron expression.

Reply:
Done